djstripe/migrations/0022_auto_20170121_1637.py

```diff
@@ -0,0 +1,153 @@
+# -*- coding: utf-8 -*-
```

Comment:
The explicit bytestrings will cause python 3 users to need another migration; please replace them.